_docs-sources/reference/services/intro/make-changes-to-your-infrastructure.md

```diff
@@ -0,0 +1,181 @@
+# Make changes to your infrastructure
+
+Now that your infrastructure is deployed, let's discuss how to make changes to it:
+
+1. [Making changes to Terraform code](#making-changes-to-terraform-code)
+1. [Making changes to Packer templates](#making-changes-to-packer-templates)
+
+### Making changes to Terraform code
+
+1. [Making changes to vanilla Terraform code](#making-changes-to-vanilla-terraform-code)
+1. [Making changes to Terragrunt code](#making-changes-to-terragrunt-code)
+1. [Making changes with Terraform Cloud or Terraform Enterprise](#making-changes-with-terraform-cloud-or-terraform-enterprise)
+
+#### Making changes to vanilla Terraform code
+
+1. **(Optional) Update your configuration**: One type of change you could do in your Terraform code (in the `.tf`
+   files) is to update the arguments you're passing to the service. For example, perhaps you update from one NAT
+   Gateway:
+
+   ```hcl
+   module "vpc" {
+     source = "git@github.com:gruntwork-io/terraform-aws-service-catalog.git//modules/networking/vpc?ref=v0.0.1"
+
+     # ... (other args ommitted for readability) ...
+     num_nat_gateways = 1
+   }
+   ```
+
+   To 3 NAT Gateways for high availability:
+
+   ```hcl
+   module "vpc" {
+     source = "git@github.com:gruntwork-io/terraform-aws-service-catalog.git//modules/networking/vpc?ref=v0.0.1"
+
+     # ... (other args ommitted for readability) ...
+     num_nat_gateways = 3
+   }
+   ```
+
+1. **(Optional) Update the Service Catalog version**: Another type of change you might make is to update the version
+   of the Service Catalog you're using. For example, perhaps you update from version `v0.0.1`:
+
+   ```hcl
+   module "vpc" {
+     source = "git@github.com:gruntwork-io/terraform-aws-service-catalog.git//modules/networking/vpc?ref=v0.0.1"
+
+     # ... (other args omitted for readability) ...
+   }
+   ```
+
+   To version `v0.0.2`:
+
+   ```hcl
+   module "vpc" {
+     source = "git@github.com:gruntwork-io/terraform-aws-service-catalog.git//modules/networking/vpc?ref=v0.0.2"
+
+     # ... (other args omitted for readability) ...
+   }
+   ```
+
+   NOTE: Whenever changing version numbers, make sure to read the [release
+   notes](https://github.com/gruntwork-io/terraform-aws-service-catalog/releases), especially if it's a backwards incompatible
+   release (e.g., `v0.0.1` -> `v0.1.0`), to avoid errors and outages (see [maintenance and
+   versioning](/docs/reference/services/intro/#maintenance-and-versioning) for details)!
```

Comment:
```suggestion
:::note

Whenever changing version numbers, make sure to read the
[release notes](https://github.com/gruntwork-io/terraform-aws-service-catalog/releases), especially if it's a backwards incompatible release (e.g., `v0.0.1` -> `v0.1.0`), to avoid errors and outages (see [maintenance and
versioning](/docs/reference/services/intro/#maintenance-and-versioning) for details)!

:::
```

Reply:
✅